tests/test_replacement.py

```diff
@@ -24,6 +24,10 @@ def test_zwj_emoji(self):
         self.assertEqual('👨‍👩‍👦', replace_colons(':man-woman-boy:'))
         self.assertEqual('👨‍🌾', replace_colons(':male-farmer:'))
 
+    def test_zwj_emoji_skin_tone(self):
+        """This tests zwj emojis that also have a skin tone"""
+        self.assertEqual('👨🏼‍🌾', replace_colons(':male-farmer::skin-tone-3:'))
```

Comment:
I think we are missing a test case here to cover all branches of the new `emoji_repl` logic. Can you rebase this branch on master (I fixed coveralls which was broken due to a dependency update) and update this test accordingly ? Aiming for 100% coverage here 😉